csharp/src/Ice/TlsServerOptions.cs

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright (c) ZeroC, Inc. All rights reserved.
+//
+
+using System;
+using System.Net.Security;
+using System.Security.Authentication;
+using System.Security.Cryptography.X509Certificates;
+
+namespace ZeroC.Ice
+{
+    /// <summary>This class represents the server side TLS configuration.</summary>
+    public class TlsServerOptions
```

Comment:
Likewise should this be a struct. Is there a reason for inheritance or references, or am I missing something?